sage.config.js

```diff
@@ -1,17 +1,13 @@
 // @ts-check
-const {sage} = require('@roots/sage');
+const Sage = require('@roots/sage');
 
-sage.when(sage.isProduction, () => {
-  sage.purge({
-    content: [
-      'resources/views/**/*',
-      'resources/assets/scripts/**/*',
-    ],
-    css: ['resources/assets/styles/**/*'],
-  });
-});
+/**
+ * Sage theme
+ */
+const theme = Sage.bootstrap();
```

Comment:
`@roots/sage` exports a `bootstrap` config and a `tailwind` config. They're wrapped so they don't conflict with another.

@Log1x to check out the one I did based on your theme, invoke `Sage.tailwind()`. I believe the rest of the file is the same between the two configs.

Config preset source for both in this dir: https://github.com/roots/bud/tree/stable/packages/sage/src/sage